### PR TITLE
fix: change message to msg when logging

### PR DESCRIPTION
**Pull request by yamii**

### CODE REVIEW

**Comment by openam** [comment]
:+1:

File: lib/req-res/status.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ module.exports = function ( options ) {
 						'data'        : mixins.cloneDeep( errorData ) || {},
 						'code'        : code,
 						'messageType' : options.messageType,
-						'message'     : mixins.cloneDeep( options.message ) || {},
+						'msg'         : mixins.cloneDeep( options.message ) || {},
```
Comment (openam) [inline lib/req-res/status.js]: Isn't the message what comes in as `data`? https://labs.omniti.com/labs/jsend

File: lib/req-res/status.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ module.exports = function ( options ) {
 						'data'        : mixins.cloneDeep( errorData ) || {},
```
Comment (openam) [inline lib/req-res/status.js]: Should we change the line above to be `log.error( 'error' , {`?

File: lib/req-res/status.js (hunk 1 above)
Comment (yamii) [inline lib/req-res/status.js]: this is the message coming from the gateway

Comment (openam) [inline lib/req-res/status.js]: It's the message from `send.error( 'message', errorObject, 'errorCode' );`

Comment (yamii) [inline lib/req-res/status.js]: or the request message

Comment (openam) [inline lib/req-res/status.js]: We might as well name them as such in the function on line 49

File: lib/req-res/status.js (hunk 2 above)
Comment (yamii) [inline lib/req-res/status.js]: :+1:

File: lib/req-res/status.js (hunk 1 above)
Comment (openam) [inline lib/req-res/status.js]: I don't know what you mean

> or the request message

Comment (yamii) [inline lib/req-res/status.js]: @openam change `msg` to `requestData`?

Comment (openam) [inline lib/req-res/status.js]: Sorry I'm an idiot. You have `errorMsg` right below this. That's what I was thinking.

Comment (yamii) [inline lib/req-res/status.js]: :) it's ok. `msg` is the requestData coming from the gateway

Comment (openam) [inline lib/req-res/status.js]: :+1:

**Pull request merged**